android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/home/HomeScreen.kt

```diff
@@ -130,16 +135,18 @@ fun HomeScreenPreview(
             avatarUrl = "https://secure.gravatar.com/avatar/8fae17b9d0c4cca18a9661bcdf650f23",
             surveys = listOf(
                 SurveyUiModel(
+                    id = "1",
                     title = "Scarlett Bangkok",
                     description = "We'd love to hear from you!",
                     coverImageUrl = "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_"
                 ),
                 SurveyUiModel(
+                    id = "2",
                     title = "ibis Bangkok Riverside",
                     description = "We'd love to hear from you!",
                     coverImageUrl = "https://dhdbhh0jsld0o.cloudfront.net/m/287db81c5e4242412cc0_"
                 )
             )
-        )
+        ) {}
```

Comment:
What will we add in this block? Do we need it?

Reply:
@blyscuit it's in a Preview function so basically just need to pass a value. We can put the default value to `onSurveyClick` but I would like to force catching the click event in implementation to not miss it (same as the default onClick event in a `Button` -> no default value)

![image](https://user-images.githubusercontent.com/16315358/210301216-fc26fe08-c80f-4046-a74d-32e3c88dc6cb.png)